language-support/java/codegen/src/ledger-tests/scala/com/digitalasset/TestUtil.scala

```diff
@@ -48,7 +48,6 @@ object TestUtil {
   )(implicit resourceContext: ResourceContext): Future[Assertion] = {
     val config = sandbox.DefaultConfig.copy(
       port = Port.Dynamic,
-      seeding = None,
```

Comment:
this seems independent of this right?

Reply:
I revert the two last commits of #7945. This is the last one that overrides the `seeding = Some(Seeding.Weak)`  from the culprit commit. I think it is fair to revert this as well.

Reply:
Makes sense :+1: